tests/test_mgxs_library_distribcell/test_mgxs_library_distribcell.py

```diff
@@ -6,29 +6,39 @@
 import hashlib
 sys.path.insert(0, os.pardir)
 from testing_harness import PyAPITestHarness
+from input_set import PinCellInputSet
```

Comment:
Ok, I see your point. I think the optimal solution would be to have an assembly geometry with replicated pin cells which produces non-zero M(D)GXS data. Is it reasonable to ask for a `AssemblyInputSet` for this PR? If not, revert back to the original `InputSet` for now and we can aim for an assembly in a future PR.